src/frontend/komponenter/Felleskomponenter/HeaderMedSøk/FagsakDeltagerkort.tsx

```diff
@@ -90,7 +90,13 @@ const FagsakDeltagerkort: React.FunctionComponent<IFagsakDeltagerkortProps> = ({
                       }`
             }
             index={index}
-            onClick={() => innloggetSaksbehandlerHarSkrivetilgang && onClick()}
+            onClick={() => {
+                if (innloggetSaksbehandlerHarSkrivetilgang) {
+                    onClick();
+                } else {
+                    deltager.fagsakId && deltager.harTilgang && onClick();
+                }
+            }}
```

Comment:
```suggestion
            onClick={() => {
                deltager.fagsakId && deltager.harTilgang && onClick();
            }}
```

Trenger man første delen av ifen her? Man kan jo ende opp i onclicken uansett om man ikke har skrivetilgang. Evt 
innloggetSaksbehandlerHarSkrivetilgang || (deltager.fagsakId && deltager.harTilgang)

Reply:
Ja, det stemmer, men tanken er at personer med skrivetilgang har alltid lov, mens for veiledere så må personen finnes. Tenkte det var bedre å gjøre det eksplisitt.